ros_speech_recognition/README.md

```diff
@@ -30,6 +30,106 @@ This package uses Python package [SpeechRecognition](https://pypi.python.org/pyp
   print result # => 'Hello, world!'
   ```
   
+If you are using `ros_speech_recognition` with `~continuous` is `True`, you can subscribe `/Tablet/voice` (`speech_recognition_msgs/SpeechRecognitionCandidates`) message.
+
+1. Launch sample launch file.
+
+
+    ```bash
+    roslaunch ros_speech_recognition sample_ros_speech_recognition.launch
```

Comment:
We may need to set `google_cloud_credentials_json:=xxx` arg?
I got the following error.
```
[ERROR] [1641453200.346079]: Unexpected error: (<class 'oauth2client.client.ApplicationDefaultCredentialsError'>, ApplicationDefaultCredentialsError('The Application Default Credentials are not available. They are available if running in Google Compute Engine. Otherwise, the environment variable GOOGLE_APPLICATION_CREDENTIALS must be defined pointing to a file defining the credentials. See https://developers.google.com/accounts/docs/application-default-credentials for more information.',), <traceback object at 0x7f980f9a6960>)
```

I think 
```
roslaunch ros_speech_recognition sample_ros_speech_recognition.launch google_cloud_credentials_json:=xxx.json
```
is more helpful.